Sources/SwiftDocC/Semantics/Metadata/CallToAction.swift

```diff
@@ -178,6 +197,7 @@ extension CallToAction {
 extension CallToAction.Purpose {
     /// The label that will be applied to a Call to Action with this purpose if it doesn't provide
     /// a separate label.
+    @available(*, deprecated, message: "Replaced with 'CallToAction.buttonLabel(for:)'.")
     public var defaultLabel: String {
```

Comment:
should this internally call `defaultLabel(for: nil)`?

Reply:
Good idea: https://github.com/apple/swift-docc/pull/566/commits/c83c4a35f0b53a508772e1e4f041722141516564.